src/commands/viewGitHubRepo.ts

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from 'vscode-azureextensionui';
+import { ext } from '../extensionVariables';
+import { EnvironmentTreeItem } from '../tree/EnvironmentTreeItem';
+import { openUrl } from '../utils/openUrl';
+
+export async function viewGitHubRepo(context: IActionContext, node?: EnvironmentTreeItem): Promise<void> {
```

Comment:
package.json says `viewItem =~ /azureStatic(WebApp|Environment)$/`, which makes me think the type for `node` should be more than just `EnvironmentTreeItem `

Reply:
Actually, the package.json is wrong.  I still get confused since I changed WebApp to be the overall app, but production is actually an environment.